molecule/default/molecule.yml

```
@@ -0,0 +1,20 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: instance
```

```suggestion
  - name: local-accounts
```
Doesn't matter, but this is in line with the other roles